.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,10 @@
+name: Auto approve
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: chris48s/approve-bot@1.0.0
```

Comment:
I made this repo under my own namespace because I don't have permission to create new repos under the @badges org, but I'd be happy to transfer it to the @badges org if someone can accept the transfer if we adopt this approach.